docs/mp/jwt.adoc

```diff
@@ -25,8 +25,27 @@
 :feature-name: JWT Authentication
 :microprofile-bundle: true
 :rootdir: {docdir}/..
+:spec-version: 1.2
```

Comment:
Use spec version defined in attributes.adoc

Reply:
fixed